me.broken_by.PdfMetadataEditor.yml

```
@@ -0,0 +1,38 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - "/app/bin/pmedit-cli"
```

```suggestion
  - "pmedit-cli"
```

Not an absolute path.

---

changed accordinly